static/js/settings_notifications.js

```diff
@@ -46,6 +52,22 @@ function rerender_ui() {
     } else {
         unmatched_streams_table.css("display", "table-row-group");
     }
+
+    // Has to be in rerender_ui because relevant rows in table are added there
+    // therefore event handler has to be set up after every call of rerender_ui
+    unmatched_streams_table.find(".unmute_stream").on("click", (e) => {
```

Comment:
We generally like to avoid having click handlers generated more than once; I pushed a proposed revision (just a single global click handler).

Reply:
Yeah, that's definitely a better solution